src/gluonts/mx/trainer/_base.py

```diff
@@ -344,6 +344,11 @@ def loop(
                             best_epoch_info["params_path"], self.ctx
                         )
 
+                logger.info(
```

Comment:
I had removed this part after we introduced model averaging. I am not sure how informative it to add the final loss info if it is does not represent the loss of the final model. It may be misleading.

Also not sure if going all the way and print the losses of all the checkpoints that we average would be informative.

Reply:
Oh, I see. But what if model_averaging is not enabled?

Reply:
There is always model averaging (at least defined). The equivalent of "not enabled" is to average only one model which is effectively selecting the best (unless the averaging strategy does something weird and does not select the best for a particular reason). In this case the final loss would be meaningful. 

I guess a way to include this is to check the num_models in the averaging strategy (see constructor of trainer) and if it is 1 you can log the loss and otherwise a message of the form "FInal loss: average over K model using X averaging strategy". Just a suggestion...

Reply:
@benidis's suggestion sounds like a transparent compromise!